core-blocks/paragraph/index.js

```diff
@@ -258,7 +258,7 @@ class ParagraphBlock extends Component {
 								setAttributes( {
 									content: before,
 									blockFootnotes: parseFootnotesFromContent( before ),
-								} );
+								}, false );
```

Comment:
Reading this code, it's hard for me to understand the purpose of the second argument, it’s just a lonely `false`.

Reply:
Also, why wouldn't we want to update the blocks visibility in this specific case?

Reply:
100% agree on this, I don't like passing booleans as parameters either. I refactored a bit this code to optimize it and make it easier to understand in 089dd9dc4e806aac976651f78283b8723a1e46c5.